src/main/java/models/Card.java

```diff
@@ -12,14 +12,17 @@
 public class Card implements Serializable {
     public final int value;
     public final Suit suit;
+    public final String name;
```

Comment:
So we're changing the value from an int to a string? This will change things a bit.